src/setup-ros-linux.ts

```diff
@@ -101,11 +105,13 @@ export async function runLinux() {
 		"-c",
 		`echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list`,
 	]);
-	await utils.exec("sudo", [
-		"bash",
-		"-c",
-		`echo "deb http://packages.ros.org/ros2/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros2-latest.list`,
-	]);
+	if(use_ros2_testing) {
```

Comment:
sorry - i think this is still not right - you don't need `if`, in this case. If use_ros2_testing is false, we should still be adding the packages.ros.org url to our apt lists. That's probably why the tests are failing